### DISCUSSION TITLE
Suggests Windows DPI scale action.

### DISCUSSION THREAD

**Discussion by duk6046** [Codectory/AutoActions#88] [Ideas]
it's very necessary for 4k projector or some situations.

**Comment by CaveWaverider** [comment]
Agreed, a Windows DPI/Font Scaling option to set per application would be great. In addition, being able to add certain scaling % to the Actions menu to switch quickly would be helpful as well.

**Comment by Anatharias** [comment]
While this would be the best, I've found out that you can trigger a DPI change by using a powershell script:

```
# $scaling = 0 : 100% (default)
# $scaling = 1 : 125% 
# $scaling = 2 : 150% 
# $scaling = 3 : 175% 
# $scaling = 4 : 200% 
param($scaling = 0)
$source = @’
[DllImport("user32.dll", EntryPoint = "SystemParametersInfo")]
public static extern bool SystemParametersInfo(
                 uint uiAction,
                 uint uiParam,
                 uint pvParam,
                 uint fWinIni);
‘@
$apicall = Add-Type -MemberDefinition $source -Name WinAPICall -Namespace SystemParamInfo –PassThru
$apicall::SystemParametersInfo(0x009F, $scaling, $null, 1) | Out-Null
```

I launch it using this command, to get a 200% scaling:
`C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -ExecutionPolicy Bypass -File "C:\Users\Anatharias\Applications\Utilitaires\QuickChangeResolution_64bit_PORTABLE\scale.ps1" 4`

**Reply by duk6046** [reply]
It's pretty useful, thank you
